task/GTFSTest.js

```diff
@@ -0,0 +1,95 @@
+const fs = require('fs');
+const fse = require('fs-extra');
+const exec = require('child_process').exec;
+const through = require('through2');
+const gutil = require('gulp-util');
+const col = gutil.colors;
+const {hostDataDir, dataDir} = require('../config');
+const {postSlackMessage} = require('../util');
+
+/**
+ * Builds an OTP graph with gtfs file. If the build is succesful we can trust
+ * the file is good enough to be used.
+ */
+function testGTFS(gtfsFile, quiet=false) {
+  let lastLog=[];
+
+  const p = new Promise((resolve, reject) => {
+    if(fs.existsSync(gtfsFile)) {
```

Comment:
if( somethings_fine ) {
<insert lots of code>
else {
 <handle error>
}

could be replace with 
if( somethings_wrong ) {
 <handle error>
 return;
}
<insert lots of code>



For readability